.github/workflows/amd.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Install pytorch
         run: |
-          pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/rocm5.1.1
+          pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/rocm5.3
```

Comment:
This line causes the following version of torch to be installed on our CI nodes:
`1.13.1+cu117`

It doesn't look like torch has released wheels compiled with 5.3 - is this a mistake?

Reply:
Hi @mrwyattii, you are correct, rocm 5.3 is installing torch version `1.13.1+cu117`. Where as 5.1.1 installs torch version `1.13.1+rocm5.1.1`. 

@jithunnair-amd, should we not proceed with this PR? Please advice.

Reply:
Hey folks, if you're looking for the latest wheels, those would be at: https://download.pytorch.org/whl/rocm5.4.2/. Is that what you want?

Reply:
lol, just saw this is an old PR from our side. But yes, if we want to update ROCm version used, it would be 5.4.2. We can go ahead and update the PR. cc @rraminen